src/modules/Modal/ModalActions.js

```diff
@@ -37,4 +38,6 @@ ModalActions.propTypes = {
   className: PropTypes.string,
 }
 
+ModalActions.create = createShorthandFactory(ModalActions, children => ({ children }))
```

Comment:
Because there can be many buttons, we need to allow the user to pass a `key` or our `childKey` prop.  The third parameter to a`createShorthandFactory ` function is a boolean telling the factory to accept keys:

```jsx
createShorthandFactory(ModalActions, children => ({ children }), true)
```

Next, this should probably map its value to an `actions` prop rather than `children`.  The render method will then map over the `actions` and create buttons:

```jsx
createShorthandFactory(ModalActions, actions => ({ actions }), true)
```

This enables users to use the `actions` prop on the `Modal.Actions` themselves as well:
```jsx
<Modal>
  <Modal.Actions actions={['No', { primary: true, content: 'Yes' }]} />
</Modal>
```